diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 0f9b1ce..149f627 100644
--- a/libsolve/src/CMakeLists.txt
+++ b/libsolve/src/CMakeLists.txt
@@ -27,7 +27,7 @@ include(ExternalProject)
 # cryptominisat
 ExternalProject_Add(cmsat
     GIT_REPOSITORY https://github.com/msoos/cryptominisat.git
-    GIT_TAG 5.6.8
+    GIT_TAG 5.11.21
 	CMAKE_GENERATOR "Unix Makefiles"
     CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DSTATICCOMPILE=ON -DNOM4RI=ON
     BUILD_COMMAND make
